src/frontend/utils/mappingTilKontrakt/andreUtbetalingsperioder.ts

```diff
@@ -0,0 +1,75 @@
+import {
+    hentUtbetalingsperiodeSpørsmålIder,
+    UtbetalingerSpørsmålId,
+} from '../../components/Felleskomponenter/UtbetalingerModal/spørsmål';
+import { ISøknadsfelt } from '../../typer/kontrakt/generelle';
+import { IUtbetalingsperiodeIKontraktFormat7 } from '../../typer/kontrakt/v7';
+import { IUtbetalingsperiode } from '../../typer/perioder';
+import { hentTekster, landkodeTilSpråk } from '../språk';
+import { sammeVerdiAlleSpråk, verdiCallbackAlleSpråk } from './hjelpefunksjoner';
+
+export const tilIAndreUtbetalingsperioderIKontraktFormat = ({
+    periode,
+    periodeNummer,
+    tilbakeITid,
+    gjelderAndreForelder,
+    erAndreForelderDød,
+}: {
+    periode: IUtbetalingsperiode;
+    periodeNummer: number;
+    tilbakeITid: boolean;
+    gjelderAndreForelder: boolean;
+    erAndreForelderDød: boolean;
+}): ISøknadsfelt<IUtbetalingsperiodeIKontraktFormat7> => {
+    const { fårUtbetalingNå, utbetalingLand, utbetalingFraDato, utbetalingTilDato } = periode;
+
+    return {
+        label: hentTekster('felles.flereytelser.periode', {
+            x: periodeNummer,
+        }),
+        verdi: sammeVerdiAlleSpråk({
+            fårUtbetalingNå: {
+                label: hentTekster(
```

Comment:
Samme kommentar som de andre. Kan vi dra det ut? 😊 tror det sparer oss for en del linjer